### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by Greg-21**
In this pull request, I updated the Polish translation to reflect all the changes that have been made over the past few months.

### CODE REVIEW

File: Common/src/main/resources/assets/jei/lang/pl_pl.json [hunk 1]
```diff
@@ -41,42 +40,53 @@
...
-  "key.jei.cheatItemStack2": "Zcheatuj 1 stos przedmiotów",
-  
-  "jei.key.category.mouse.hover": "JEI (najeżdżanie myszą)",
+  "key.jei.cheatItemStack": "Zcheatuj 1 stack",
+  "key.jei.cheatItemStack2": "Zcheatuj 1 stack",
+
+  "jei.key.category.hover.config.button": "JEI (najechanie kursorem na przycisk konfiguracji)",
+  "key.jei.toggleCheatModeConfigButton": "Włącz/wyłącz tryb cheatów",
+
+  "jei.key.category.edit.mode": "JEI (tryb edycji)",
```
Comment (Greg-21) [inline Common/src/main/resources/assets/jei/lang/pl_pl.json:70]: What do you think about renaming "Edit Mode" ("tryb edycji" here in Polish) to "Hide Ingredients Mode" in the source U.S. English translation file, as the main purpose of this mode is to hide/show ingredients and nothing else?

Comment (mezz) [reply Common/src/main/resources/assets/jei/lang/pl_pl.json:70]: Yes I think it is only used to hide or show ingredients. That sounds fine to me.

Comment (Greg-21) [reply Common/src/main/resources/assets/jei/lang/pl_pl.json:70]: Would you prefer that I create another commit and include it in this pull request, or should I open a completely new pull request instead?

Comment (mezz) [reply Common/src/main/resources/assets/jei/lang/pl_pl.json:70]: I can merge this one and you can create a new pull request. That way these changes can be released now.

**Pull request merged**